platform/rct/Libraries/CMSIS/Device/ST/STM32F30x/Source/Templates/system_stm32f30x.c

```diff
@@ -201,11 +201,6 @@ void SystemInit(void)
        AHB/APBx prescalers and Flash settings ----------------------------------*/
     SetSysClock();
 
-#ifdef VECT_TAB_SRAM
-    SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM. */
-#else
-    SCB->VTOR = FLASH_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal FLASH. */
```

Comment:
Sorry, been busy.  VTOR points to the beginning of the application image.  In this case, it's 0x08004000 (16k in from the start of flash).  We set the location of VTOR in the CPU init hal code.

By default, this comes up as 0x08000000 on any reset.